container-health-check.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#echo "Usage: ${0} <URL> <ClientCertLocation> <ClientKeyLocation> <CaCertLocation>"
```

Comment:
Usage doesn't match the spec from the ticket. Should use parameter names instead of assuming an order.